docs/articles/documentation/using-testcafe/installing-testcafe.md

```diff
@@ -6,7 +6,14 @@ checked: false
 ---
 # Installing TestCafe
 
-You can install TestCafe globally or locally in your project.
+You can install TestCafe through `npm` globally or locally in your project.
+
+* [Globally](#globally)
+* [Locally](#locally)
+
+Alternatively, use a Docker image with TestCafe preinstalled.
```

Comment:
Why we need this remark? I believe just "Using TestCafe Docker Image" list item will work pretty well.